docs/user-guide.md

```diff
@@ -19,9 +19,23 @@ Below are simple steps to use this portal:
 
 {% embed url="https://www.youtube.com/playlist?list=PLJH-POb_55z8YYS_qAk_QNBQeiQ2VrtZD" %}
 
+## Compliance Toolkit Landing Page
+
+On the CTK landing page, partners can view the introduction of the CTK portal and watch a video about CTK. Additionally, they can login or register by clicking the respective buttons. The landing page also provides links to other resources such as the GitHub repository, documentation, and partner details.
+
+![CTK Landing Page1](\_images/ctk-landing-page1.png)
+
+Also, it highlights some of CTK's powerful features, like its pre-built test cases, validations, and detailed reporting.
+
+![CTK Landing Page2](\_images/ctk-landing-page2.png)
+
+The landing page provides information about the SBI, Android SBI, SDK, and ABIS specifications. Additionally, partners can download the compliance toolkit APK by selecting the download option.
```

Comment:
The landing page offers details about the SBI, Android SBI, SDK, and ABIS specifications. Furthermore, partners have the option to download the compliance toolkit APK by choosing the download button.